packages/contracts-bedrock/scripts/DeployOPChain.s.sol

```diff
@@ -72,6 +80,16 @@ contract DeployOPChainInput is BaseDeployIO {
             _l2ChainId = _value;
         } else if (_sel == this.gasLimit.selector) {
             _gasLimit = SafeCast.toUint64(_value);
+        } else if (_sel == this.disputeGameType.selector) {
+            _disputeGameType = GameType.wrap(SafeCast.toUint32(_value));
+        } else if (_sel == this.disputeMaxGameDepth.selector) {
+            _disputeMaxGameDepth = _value;
+        } else if (_sel == this.disputeSplitDepth.selector) {
+            _disputeSplitDepth = _value;
```

Comment:
Safe cast to uint64 for these, that way their size is also enforced in soldity tests?

Reply:
https://github.com/ethereum-optimism/optimism/pull/12228/commits/6fa9a8d9771a30f734bd69e269583e04850346db